process/validators/serialize.go

```diff
@@ -32,8 +33,11 @@ func (vp *validatorsProcessor) SerializeValidatorsRating(
 ) ([]*bytes.Buffer, error) {
 	buffSlice := data.NewBufferSlice()
 
+	// from elrond-go index is "shardID_epoch" - to keep backwards compatibility have to change here
```

Comment:
```
// inside elrond-go, the index is "shardID_epoch" so in order to keep backwards compatibility some adjustments have to be made.
// shardID from index name has to be removed because it is sufficient to have document id = blsKey_epoch
```

Reply:
done.